app/dawn-hub/Views/OnboardingViews/Step3_ChooseDefaultSearchView/ChooseDefaultSearchView.jsx

```diff
@@ -78,6 +78,24 @@ class ChooseDefaultSearchView extends Component {
 		document.addEventListener('click', this.handleClickAway);
 
 		this.fetchSearchEnginesAsync();
+
+		const { setupLifecycle: { searchSetupSeen } } = this.props;
+
+		if (searchSetupSeen) {
+			const { defaultSearch } = this.props;
+			const searchChoices = [
+				SEARCH_GHOSTERY,
+				SEARCH_BING,
+				SEARCH_YAHOO,
+				SEARCH_STARTPAGE,
+			];
+			const isOther = searchChoices.reduce((accum, choice) => {
+				if (accum === false) return false;
+				return choice !== defaultSearch;
+			}, false);
```

Comment:
You can simplify to
`const isOther = searchChoices.includes(defaultSearch)`

(Also I think your version will always return `false`? Since `accum` is initialized to `false` and so the condition in line 93 is always satisfied. Bigger picture, with the simpler `includes` version it's much easier to be confident that the code does what you intend)

Reply:
Yea, you're right. Definitely messed up on the logic here, and `.includes()` is a much simpler implementation. I changed the check to `const isOther = !searchChoices.includes(defaultSearch)` as `.includes(defaultSearch)` returning `true` would mean that `isOther` should be `false`.

Reply:
Great, thanks man!